photo-points/Models/Capture.cs

```diff
@@ -11,13 +11,10 @@ public class Capture
     public long captureID { get; set; }
     public byte photo { get; set; }
     public DateTime captureDate { get; set; }
-    public long photoPointID { get; set; }
-    public long userID { get; set; }
+    
+    public ICollection<Data> data { get; set; }
```

Comment:
we suppose to have approved,

Reply:
Added in now! Thanks for catching this. Accidentally deleted when I was adding the Navigation properties.